### DISCUSSION TITLE
Return failed transaction

### DISCUSSION THREAD

**Discussion by lucker28** [1200wd/bitcoinlib#390] [Q&A] [answered]
I sent [transaction](https://www.blockchain.com/explorer/transactions/btc/669b803bc0b964cb6b30ab257d238d920c868617093beb00d53ec619257a71e9) from bitcoinlib and it stucked with message: "Transaction orphaned".

Any chance to cancel it and resend?

Code which I used:

```
w = Wallet('btcwallet')
t = w.send([(btcaddress, satoshi)], offline=False, fee='normal')
```

**Comment by mccwdev** [comment, marked as answer]
It looks like the transaction is correct but pays insufficient fees.

You can not cancel a transaction, however you could:
* Resend the same transaction with a higher fee, however replace-by-fee was not enabled with this transaction.
* Create a new transaction which spends one of the outputs of this transaction. See Child pays for parent: https://bitcoinops.org/en/topics/cpfp/
* Wait till the transactions is dropped from the mempool, but this can take up to two weeks.

**Reply by lucker28** [reply]
Thx a lot.
